### ISSUE TITLE
General Use Queries

### ISSUE THREAD

**Issue by MarcoAZ** [meynmd/cs340#12]
Project requirements dictate that all queries used on the website should be collected in a single file and inputs displayed like so:
> SELECT salary FROM employee WHERE salary > [salaryInput ]; 
Now might be a good time to collect the queries we've written so far and add new queries to this file as we move along.

**Comment by meynmd** [comment]
Yes, I've started doing that because I thought we might as well show the queries we had so far to the instructors. (They offered "feedback", remember?) So I have a list going, which I can add to the repo.

**Comment by meynmd** [comment]
I added the list as generalQueriesList.txt

**Comment by meynmd** [comment]
I split the list into "create queries" and "general use queries" because those are listed separately as categories in the assignment. I also tried to update the list as best I could--it's getting hard to keep track of what's in all the php files we have!

**Comment by MarcoAZ** [comment]
Yeah, it's gotten pretty lengthy!
